static/app/components/nav/sidebar.tsx

```diff
@@ -0,0 +1,187 @@
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+import {motion} from 'framer-motion';
+
+import Feature from 'sentry/components/acl/feature';
+import Link from 'sentry/components/links/link';
+import {useIndicator} from 'sentry/components/nav/useIndicator';
+import type {SidebarItem} from 'sentry/components/nav/utils';
+import {
+  getActiveProps,
+  getActiveStatus,
+  useLocationDescriptor,
+} from 'sentry/components/nav/utils';
+import {space} from 'sentry/styles/space';
+import theme from 'sentry/utils/theme';
+import {useLocation} from 'sentry/utils/useLocation';
+
+export const Sidebar = styled('div')`
+  height: 40px;
+  width: 100vw;
+  padding: ${space(2)} 0;
+  border-right: 1px solid ${theme.translucentGray100};
+  background: #3e2648;
+  background: linear-gradient(180deg, #3e2648 0%, #442c4e 100%);
+  display: flex;
+  flex-direction: column;
+  z-index: ${theme.zIndex.sidebar};
+
+  @media screen and (min-width: ${p => p.theme.breakpoints.medium}) {
+    height: unset;
+    width: 74px;
+  }
+`;
+
+function Items({children}) {
+  const {indicatorProps, containerProps} = useIndicator();
+
+  return (
+    <Fragment>
+      <Indicator {...indicatorProps} />
+      <ItemList {...containerProps}>{children}</ItemList>
+    </Fragment>
+  );
+}
+
+const ItemList = styled('ul')`
+  position: relative;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+  padding-top: ${space(1)};
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  color: rgba(255, 255, 255, 0.85);
+
+  @media screen and (min-width: ${p => p.theme.breakpoints.medium} {
+    gap: ${space(1)};
+  }
+`;
+
+function Item({
+  to,
+  label,
+  icon,
+  submenu,
+  check,
```

Comment:
Is there a way we can use the same signature as the Feature component, since that is a signature people folks are already familiar with?

Reply:
Resolved in [a9c0f1a](https://github.com/getsentry/sentry/pull/77665/commits/a9c0f1aac5788b60053556463c6669a4bbcf4fe7)